src/main/java/com/rocketden/main/dto/game/TesterResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.rocketden.main.dto.game;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+// This class is mapped directly to the tester service's RunDto class.
+public class TesterResponse {
+    private Boolean status;
+    private String output;
+}
```

Comment:
I want to note that I think the `TesterResponse` will end up being different than this because I'm predicting that we do the process of parsing the `output` and gathering the results for each test, errors, and # correct -- and passing that through here. So I can envision us essentially returning a more complete Submission object, which means the TODO shown [here](https://github.com/rocketden/main/pull/100/files#diff-df30ccd300c4cf9576689b18935e50bcc164e2387d5485a5c3963827c86f5532R123) in my mind is really more relevant for the tester repo.

Reply:
Makes sense yeah - I can imagine most of that will be done on tester but there might still be a few missing pieces (potentially isSolved) to clean up here (also having a separate response class might be useful for the same reasoning behind our current DTO/model structure